### PR TITLE
Fibonacci - Darth

### PR DESCRIPTION
**Pull request by Morfiniusz**
:)

### CODE REVIEW

**Comment by ziobron** [comment]
✅ CI Passed - Task: fibonacci
🏅 Granted 6 XP for @Morfiniusz

**Comment by ziobron** [comment]
⏱ 3 XP extra for @Morfiniusz for on-time delivery

File: homework/fibonacci/fibonacci.hpp [hunk 1]
```diff
@@ -1,11 +1,35 @@
...
-    // TODO: Your implementation goes here
+   int first = 0;
+   int second = 1;
+   if(sequence == 0){
+       return 0;
+   }
+   else if (sequence == 1){
+       return 1;
+   }
+   else if (sequence >= 2){
```
Comment (nauka-programowania-MA) [inline homework/fibonacci/fibonacci.hpp:12]: Sequence hold exactly the same value as you want to return for 0 and 1 :)

Better write:

`if (sequence < 2) {
    return sequence;
}`

You avoid second if

Comment (nauka-programowania-MA) [inline homework/fibonacci/fibonacci.hpp:12]: If you return variable here, you don't need to use `else if`

File: homework/fibonacci/fibonacci.hpp [hunk 2]
```diff
@@ -1,11 +1,35 @@
...
+       return first;
+   }  
     return 0;
 }
 
 int fibonacci_recursive(int sequence) {
-    // TODO: Your implementation goes here
+        if(sequence == 0){
+        return 0;
+    }
```
Comment (nauka-programowania-MA) [inline homework/fibonacci/fibonacci.hpp:27]: Wrong indentation

File: homework/fibonacci/fibonacci.hpp [hunk 3]
```diff
@@ -1,11 +1,35 @@
...
+    else if (sequence == 1){
+        return 1;
+    }
+    else{
```
Comment (nauka-programowania-MA) [inline homework/fibonacci/fibonacci.hpp:31]: If you return variable here, you don't need to use `else if` or `else`

**Review by nauka-programowania-MA — APPROVED** [review]

**Pull request closed without merging**